packages/fields-document/src/test-fixtures.js

```diff
@@ -0,0 +1,85 @@
+import { getItems } from '@keystonejs/server-side-graphql-client';
+import { Text } from '@keystonejs/fields';
+import { DocumentField } from './index';
+
+const DOC1 = [{ type: 'paragraph', children: [{ text: 'other', bold: true }] }];
+const DOC2 = [{ type: 'paragraph', children: [{ text: 'FOOBAR', bold: true }] }];
+const DOC3 = [{ type: 'paragraph', children: [{ text: 'fooBAR', bold: true }] }];
+const DOC4 = [{ type: 'paragraph', children: [{ text: 'foobar', bold: true }] }];
+
+export const name = 'DocumentField';
+export const type = DocumentField;
+export const exampleValue = () => JSON.stringify(DOC1);
+export const exampleValue2 = () => JSON.stringify(DOC2);
+export const supportsUnique = true;
+export const fieldName = 'testField';
+
+export const getTestFields = () => ({ name: { type: Text }, testField: { type } });
+
+export const initItems = () => {
+  return [
+    // An empty string is not a correct DocumentField schema structure.
+    // We have a pending TODO in the document field's implementation to revisit this.
+    { name: 'a', testField: '' },
+    { name: 'b', testField: JSON.stringify(DOC1) },
+    { name: 'c', testField: JSON.stringify(DOC2) },
+    { name: 'd', testField: JSON.stringify(DOC3) },
+    { name: 'e', testField: JSON.stringify(DOC4) },
+    { name: 'f', testField: null },
+    { name: 'g' },
+  ];
+};
+
+export const storedValues = () => [
+  // Updating document value with invalid string (e.g empty string) results in null.
+  { name: 'a', testField: null },
+  { name: 'b', testField: JSON.stringify(DOC1) },
+  { name: 'c', testField: JSON.stringify(DOC2) },
+  { name: 'd', testField: JSON.stringify(DOC3) },
+  { name: 'e', testField: JSON.stringify(DOC4) },
+  { name: 'f', testField: null },
+  { name: 'g', testField: null },
+];
+
+export const supportedFilters = [
+  'null_equality',
+  'equality',
+  'equality_case_insensitive',
+  'in_empty_null',
+  'in_value',
+];
+
+// Document field type doesn't support `start_with` and `ends_with`
```

Comment:
Hmm, I wonder if we want to be supporting the other string operations if we don't support these two? I'd like to have a discussion about this before we ship this.

Reply:
This is because a document field value, being a string though, needs to have certain schema [interface](https://docs.slatejs.org/concepts/01-interfaces). 
```ts
interface Element {
  children: Node[]
}
```
And in our GraphQL filtering, we may be only interested in the text content which is: 
```ts
interface Text {
  text: string
}
```
So, given a unique schema,`contains` provides a value while filtering whereas `starts_` and `ends_` variants won't make much of a sense.